app/models/plugins/survey.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+class Plugins::Survey < ActiveRecord::Base
+  has_many :forms, -> { order(position: :asc, created_at: :asc) }, as: :formable, dependent: :destroy
+
+  belongs_to :page, touch: true
+  after_create :ensure_required_fields
+
+  DEFAULTS = {}.freeze
```

Comment:
When a plugin is created, it calls `class.new(class::DEFAULTS)`, so all plugins need to have something